src/elements/DestructiveDialog/DestructiveDialog.less

```diff
@@ -0,0 +1,32 @@
+/**
+ * Destructive Dialog styles
+ */
+
+ @ui-destructive-dialog-font: @ui-font-sans-serif;
+ @ui-destructive-dialog-letter-spacing: 0.02em;
+
+.ui-destructive-dialog-container {
+	font-family:  @ui-destructive-dialog-font;
+	letter-spacing:  @ui-destructive-dialog-letter-spacing;
+	height: 186px;
+	width: 280px;
+
+	.destructive-dialog-title {
+		font-size: 20px;
+	}
+
+	.destructive-dialog-content {
+		color: @ui-color-gray-dark;
+		font-size: 16px;
+	}
+
+	.destructive-dialog-button-wrapper {
+		position: relative;
+		left: 100px;
```

Comment:
Get rid of the `position`/`left` properties and just do a `float: right` to kick the buttons to the right instead of using a specific value. This will allow us to have an adjustable width on the overall dialog while still keeping the buttons properly placed to the right.